docs/source/examples/md/sec_agg_example.md

```diff
@@ -0,0 +1,195 @@
+# Example in Recognize Digits with DPSGD
+
+This document introduces how to use PaddleFL to train a model with Fl Strategy: Secure Aggregation. Using Secure Aggregation strategy, the server can aggregate the model parameters without learning the value of the parameters.
+
+### Dependencies
+
+- paddlepaddle>=1.6
+
+### How to install PaddleFL
+
+Please use python which has paddlepaddle installed
+
+```
+python setup.py install
+```
+
+### Model
+
+The simplest Softmax regression model is to get features with input layer passing through a fully connected layer and then compute and ouput probabilities of multiple classifications directly via Softmax function [[PaddlePaddle tutorial: recognize digits](https://github.com/PaddlePaddle/book/tree/develop/02.recognize_digits#references)].
```

Comment:
output probabilities
multiple classes?

Reply:
Thanks for your kind comments. I'm really appreciate that. I have optimize my demo according to your comments. Pls see my second commit.